pyo3-ffi/examples/sequential/noxfile.py

```diff
@@ -1,6 +1,8 @@
 import sys
 import nox
 
+nox.options.default_venv_backend = "uv"
```

Comment:
Rather than modifying in every noxfile, it might be possible to use the environment variable in `build.yml` like I tried in https://github.com/PyO3/pyo3/pull/5363/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R27

Reply:
done!